src/ResourceManagement/Compute/Microsoft.Azure.Management.Compute/project.json

```diff
@@ -1,5 +1,5 @@
 {
-  "version": "11.3.0-prerelease",
+  "version": "11.4.0-prerelease",
```

Comment:
Thanks. I think we can update the version to 12.0.0 when we update the API version to 2016-03-30.